x-pack/plugins/ml/public/ml_url_generator/dataframe_analytics_urls_generator.ts

```diff
@@ -0,0 +1,70 @@
+/*
```

Comment:
We always use two words for 'data frame' so for consistency this file should be renamed to `data_frame_analytics_urls_generator.ts`

Reply:
Renamed here 
https://github.com/elastic/kibana/pull/75696/commits/9962c25afba12c5d6ebc699f1134f010176b068f